src/core/BufferAttribute.js

```diff
@@ -32,6 +32,12 @@ function BufferAttribute( array, itemSize, normalized ) {
 
 }
 
+Object.defineProperty( BufferAttribute.prototype, "needsUpdate", {
+
+	set: function(value) { if ( value === true ) this.version ++; }
+
+});
```

Comment:
No big deal. We can fix the formatting... Getting it right the first time just makes less work for others... Thank you for your efforts.  :)